src/libs/Trie/index.js

```diff
@@ -0,0 +1,90 @@
+import _ from 'underscore';
+import TrieNode from './TrieNode';
+
+class Trie {
+    constructor() {
+        this.root = new TrieNode();
+    }
+
+    /**
+    * Add a word to the Trie
+    * @param {String} word
+    * @param {Object} [metaData]
```

Comment:
can you add comments about what these variables are (specifically metaData and containChar)

Reply:
I added comments and converted `containChar` from `string` to `boolean`, I think using `boolean` is better than `string`